### ISSUE TITLE
Indentation levels duplicate on save using Prettier/VSCode

### ISSUE THREAD

**Issue by mraichelson** [Shopify/prettier-plugin-liquid#35]
**Describe the bug**
After each save in some instances when using the Prettier plugin along with Prettier on-save in VSCode some elements have their indentation level duplicated on each save.

**Unformatted source**
```liquid
<style>section.foo { display:none; }</style>
```

**Expected output**
```liquid
<style>
  section.foo { display:none; }
</style>
```

**Actual output**

File duplicates intended indentation level and adds it to the previous indentation level on every file save.
```liquid
<!-- after a single save -->
<style>
  section.foo { display:none; }
</style>
<!-- after another save -->
<style>
    section.foo { display:none; }
</style>
<!-- after another save -->
<style>
      section.foo { display:none; }
</style>
```

**Debugging information**
 - OS: Mac
 - Version: 0.14

**Additional context**
Here are a couple of screen recordings showing this behavior.

https://user-images.githubusercontent.com/70476/180024543-4b7d9886-4a8f-4a9b-aeee-471c032ad3f9.mp4

https://user-images.githubusercontent.com/70476/180025572-3006f966-b5d3-41e2-afde-45ce07d435b1.mp4

**Comment by charlespwd** [comment]
:wave: I feel like you must have a conflicting plugin as this is not happening to me :/

https://user-images.githubusercontent.com/4990691/181823747-31680f8c-a5c5-4748-a6f4-94c8b4a551c9.mp4

We also run idempotence unit tests that should prevent this from happening.

**Comment by mraichelson** [comment]
I don't mean to dig this one back up since it could very well be a conflicting extension (though I purged a number of them recently and things have been working as intended). 

But, after having things working properly for a while I tried setting my `.prettierrc` to use tabs instead of spaces for indentation and when `useTabs` is `true` I did see indentation going weird again the same as before across two different machines. 

Setting it back to `false` (the default option, but [that may potentially change](https://github.com/prettier/prettier/issues/7475) in the future) makes the indentation go back to normal.
